networking/metallb/metallb-upgrading-operator.adoc

```diff
@@ -10,7 +10,7 @@ The automatic upgrade procedure does not work as expected from {product-title} 4
 
 . Delete the previously installed Operator version for example 4.10. Ensure that the namespace and the `metallb` custom resource are not removed.
```

Comment:
@xenolinux Do you want to clearly state that there are no issues upgrading from 4.11 to 4.12, as you are saying "Delete the previously installed Operator version **for example** 4.10"?

Reply:
Addressed